mungegithub/bulk-lgtm/www/script.js

```diff
@@ -0,0 +1,110 @@
+"use strict";
+var app = angular.module('BulkLGTMModule', ['ngMaterial', 'angularMoment']);
+
+app.config(['$compileProvider', function ($compileProvider) {
+  $compileProvider.debugInfoEnabled(false);
+}]);
+
+app.controller('BulkLGTMCntl', ['DataService', '$interval', '$location', BulkLGTMCntl]);
+
+function BulkLGTMCntl(dataService, $interval, $location) {
+  var self = this;
+  self.prs = {};
+  self.prsCount = 0;
+  self.health = {};
+  self.metadata = {};
+  self.testResults = {};
+  self.tabLoaded = {};
+  self.functionLoading = {};
+  self.queryNum = 0;
+  self.selected = 1;
+  self.OverallHealth = "";
+  self.StatOrder = "-Count";
+  self.location = $location;
+  self.lgtm = lgtm;
+
+  dataService.getData("bulkprs/prs").then(function success(response) {
+    self.prs = response.data;
+  }, function error(response) {
+    console.log("Error: Getting pr data: " + response);
+  });
+
+  // Request Avatars that are only as large necessary (CSS limits them to 40px)
+  function fixPRAvatars(prs) {
+    angular.forEach(prs, function(pr) {
+      if (/^https:\/\/avatars.githubusercontent.com\/u\/\d+\?v=3$/.test(pr.AvatarURL)) {
+        pr.AvatarURL += '&size=40';
+      }
+    });
+  }
+
+  function refreshPRs() {
```

Comment:
Is this used at all? Doesn't seem so. Which means fixPRAvatars is also a vestigial appendage?